internal/ingress/annotations/mirror/main.go

```diff
@@ -41,18 +69,20 @@ func NewParser(r resolver.Resolver) parser.IngressAnnotation {
 // ParseAnnotations parses the annotations contained in the ingress
 // rule used to configure mirror
 func (a mirror) Parse(ing *networking.Ingress) (interface{}, error) {
-	config := &Config{}
-	var err error
-
-	config.URI, err = parser.GetStringAnnotation("mirror-uri", ing)
-	if err != nil {
-		config.URI = ""
+	config := &Config{
+		URI: fmt.Sprintf("/_mirror-%v", ing.UID),
```

Comment:
I see below that mirroring is configured per server block, which can correspond to multiple Ingress resources. We should somehow make this server specific not ingress.

Reply:
> I see below that mirroring is configured per server block, which can correspond to multiple Ingress resources

To all paths defined in an ingress. 

> We should somehow make this server specific not ingress.

Not sure I follow

Reply:
Let's say there are 3 ingresses defining 3 paths for `example.com` server. When I enable mirroring on first ingress, then the internal path will be created based on the first ingress's UID. While describing this I realized it is fine :)